### ISSUE TITLE
[TORCH] Support NLL loss with Gaussian distribution of target

### ISSUE THREAD

**Issue by Prakhar-Dixit** [llvm/torch-mlir#4234]
**GaussianNLLLoss Function:**

https://docs.pytorch.org/docs/stable/generated/torch.nn.GaussianNLLLoss.html#gaussiannllloss
Currently this op is not supported in torch-mlir.

**Comment by Prakhar-Dixit** [comment]
I am interested in implementing this op. Could you please assign it to me ?
@vivekkhandelwal1 @zjgarvey

**Comment by Prakhar-Dixit** [comment]
This op doesn't have support in the PyTorch registry and hence cannot be traced. Could you please close this issue.
